context/app/static/js/components/workspaces/hooks.ts

```diff
@@ -264,7 +278,15 @@ export function useCreateAndLaunchWorkspace() {
   const { toastError, toastSuccess } = useSnackbarActions();
 
   const createAndLaunchWorkspace = useCallback(
-    async ({ body, templatePath }: { body: CreateWorkspaceBody; templatePath: string }) => {
+    async ({
+      body,
+      templatePath,
+      resourceOptions,
+    }: {
+      body: CreateWorkspaceBody;
+      templatePath: string;
+      resourceOptions: WorkspaceResourceOptions;
+    }) => {
```

Comment:
We can optionally define this as a type (`CreateWorkspaceArgs` or similar) to avoid the inline type definition.